tfhe/src/core_crypto/algorithms/test/mod.rs

```diff
@@ -194,6 +194,7 @@ pub const MULTI_BIT_2_2_2_PARAMS: MultiBitTestParams<u64> = MultiBitTestParams {
     ciphertext_modulus: CiphertextModulus::new_native(),
     grouping_factor: LweBskGroupingFactor(2),
     thread_count: ThreadCount(5),
+    tuniform: false,
```

Comment:
this is not needed you can match on the noise distribution

Reply:
Thanks, fixed it :+1: